src/components/UserSelector.tsx

```diff
@@ -1,35 +1,54 @@
 import React from 'react';
+import { User } from '../types/User';
+import { UserList } from './UserList';
 
-export const UserSelector: React.FC = () => {
+type Props = {
+  handleListDropDown: () => void;
+  selectedUser: User | null;
+  isDropDownList: boolean;
+  setIsDropDownList: (value: boolean) => void;
+  users: User[];
+  handleChoseUser: (user: User) => void;
+};
+
+export const UserSelector: React.FC<Props> = ({
+  handleListDropDown,
+  selectedUser,
+  isDropDownList,
+  setIsDropDownList,
+  users,
+  handleChoseUser,
+}) => {
   return (
-    <div
-      data-cy="UserSelector"
-      className="dropdown is-active"
-    >
+    <div data-cy="UserSelector" className="dropdown is-active">
       <div className="dropdown-trigger">
         <button
           type="button"
           className="button"
           aria-haspopup="true"
           aria-controls="dropdown-menu"
+          onClick={handleListDropDown}
+          onBlur={() => setTimeout(() => setIsDropDownList(false), 210)}
```

Comment:
Pls move the function inside `onBlur` handler to a separate const and use as a callback here